src/components/Highlight.vue

```diff
@@ -6,9 +6,8 @@
 </template>
 
 <script>
+import instantsearch from 'instantsearch.js/es';
 import { createSuitMixin } from '../mixins/suit';
-import { getPropertyByPath } from '../util/object';
```

Comment:
this helper can be removed now i think

Reply:
https://github.com/algolia/vue-instantsearch/pull/730/commits/b37ff57422dc9b091a56cf96f67f11455a53fcb6